RLTS/SendEmail/etc/config.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Store:etc/config.xsd">
+    <default>
+        <email_configure_setting>
```

Comment:
Unnecessary/inappropriate code - structure of the code is wrong ...

instead of 

```
        <email_configure_setting>
            <general>
                <email_from>usama.aslam@rltsquare.com</email_from>
            </general>
        </email_configure_setting>
        <email_configure_setting>
            <general>
                <email_from_name>Usama Aslam</email_from_name>
            </general>
        </email_configure_setting>
```

should be something like

```
<email_configure_setting>
    <general>
        <email_from>usama.aslam@rltsquare.com</email_from>
        <email_from_name>Usama Aslam</email_from_name>
    </general>
</email_configure_setting>
```